.github/labels.md

```diff
@@ -0,0 +1,252 @@
+
+#### Label `category:feature`
+
+The Issue/PR describes or implements a new game feature.  Features are parts of
+the game which a player can interact with e.g.: A new species, a new gameplay
+mechanic or a new command line switch for the server that changes gameplay
+behavior.
+
+#### Label `category:idea`
+
+??? No idea, overlaps with `category:feature` or `state:work in progress` IMO.
+
+#### Label `category:question`
+
+??? Questions don't belong into the issue tracker IMO.
+
+#### Label `category:refactoring`
+
+The Issue/PR describes or contains a new or improved implementation of an
+existing feature or any other code restructuring that doesn't modify the current
+game mechanics or representation.
+
+#### Label `category:request`
+
+??? Requests don't belong into the issue tracker IMO.
+
+#### Label `category:tweak`
+
+The PR contains insignificant code changes that don't involve features,
+refactoring or bug fixing but code style grooming or value tweaking.
+
+#### Label `component:AI`
+
+The Issue/PR deals with the python AI decision making code or affects it.
+
+#### Label `component:art music text assets`
+
+The Issue/PR deals with creating or modifying game assets or verification if the
+assets match the style goals of the game.
+
+#### Label `component:build system`
+
+The Issue/PR deals with CMake, Visual Studio, Xcode, the respective platform
+SDKs or the general build process.
+
+#### Label `component:deployment`
+
+The Issue/PR deals with the deployment process or the software used create the
+installer or other distribution packages.
+
+#### Label `component:documentation`
+
+The Issue/PR deals with Doxygen or primarily contains documentation.
+
+#### Label `component:game mechanic`
+
+The Issue/PR deals with the currently used or planned game mechanics.  The
+content of the PR may require extensive gameplay balance testing.
+
+#### Label `component:infrastructure`
+
+The Issue/PR deals with non-game related infrastructure that aids development.
+For example this includes issues with the project homepage, used external
+services like continuous integration or hosting of release binaries.
+
+#### Label `component:internal`
+
+The Issue/PR deals with any project component that has no explicit `component`
+label.
+
+#### Label `component:network`
+
+The Issue/PR deals with TCP/IP networking and establishing a test network
+infrastructure.
+
+#### Label `component:scripting`
+
+The Issue/PR deals with the FOCS language and probably game mechanics.  Probably
+most used in conjunction with `component:game mechanic` and
+`component:art music text assets`.
+
+#### Label `component:translation`
+
+The Issue/PR deals with the game style, game terminology, stringtable syntax
+and the translation process in general.  It maybe helpful to know the language
+of the translation to proofread.
+
+#### Label `component:UI`
+
+The Issue/PR deals with the game interface, the expected user experience and
```

Comment:
You have a point with your reply to my original comment about "game style", so maybe add something like: "...the game interface, consistent style in game representation,..."?